app/src/main/java/org/kiwix/kiwixmobile/main/TabsAdapter.java

```diff
@@ -45,10 +45,12 @@
   private final MainActivity activity;
   private TabClickListener listener;
   private int selectedPosition = 0;
+  private Context context;
 
-  TabsAdapter(MainActivity activity, List<KiwixWebView> webViews) {
+  TabsAdapter(MainActivity activity, List<KiwixWebView> webViews, Context context) {
```

Comment:
Activity inherits context, so you don't need to add `context` you can use `activity` instead.

Reply:
@abdulwd yeah, that's better. Done.

Reply:
Hi @abdulwd , Review this please.

Reply:
@abdulwd I made the changes. Review this please.